docker/requirements.yml

```
@@ -34,7 +34,7 @@ dependencies:
   - tqdm              # cil
   - numba             # cil
   ## - tigre          # cil (GPU) # TODO: remove second # prefix once tigre is build for numpy>=2
-  # - astra-toolbox 2 cuda* # cil (GPU)
+  # - astra-toolbox=2.1=cuda* # cil (GPU)
```

2.1 might not work; best to revert this


```suggestion
  # - astra-toolbox 2 cuda* # cil (GPU)
```